### ISSUE TITLE
Is it possible to play against bots using the openbw engine?

### ISSUE THREAD

**Issue by dukebw** [OpenBW/openbw#15]
Hello, I was wondering if it is intended and already implemented for a human player to play against bots, using the OpenBW engine. I am able to use the OpenBW/bwapi fork of BWAPI to start a game, but there is no user interface and no way to control the units. Thanks!

**Comment by shedatc** [comment]
Playing against bots would be great, especially if this bots was design to train the humans :smiley:

**Comment by dgant** [comment]
OpenBW only supports replays and bot-on-bot play at the moment. Fortunately Brood War is now free :) and you can play it on Linux using Wine.

**Comment by shedatc** [comment]
You're right about the fact that OpenBW is not required to play BW, even on GNU/Linux. I personally run Remastered via Wine and it run perfectly.

But, still, what is missing from OpenBW to get the possibility to fight against an AI? Is it just about enriching the user input handling of the UI component?

**Comment by tscmoo** [comment]
What is missing? Well, the entire UI for one. There could also be quite a bit of code required related to handling of user input, and figuring out which actions to input when you click here and there.
Also, as mentioned, I don't really see a great need for it, so there are no plans for it right now

**Comment by bftjoe** [comment]
Also blizzard would probably send a cease and desist since they are selling remastered.
